src/index.ts

```diff
@@ -1,17 +1,115 @@
+import {
+  getHubApiUrl,
+  getPortalApiUrl,
+  getSiteById,
+  IDomainEntry,
+  IHubRequestOptions,
+  lookupDomain,
+} from '@esri/hub-common';
+import { IContentSearchRequest } from '@esri/hub-search';
+import { Request, Response } from 'express';
+import * as _ from 'lodash';
 import { version } from '../package.json';
+import { getDataStreamDcatAp201 } from './dcat-ap';
+import * as config from 'config';
 
 export = class Output {
-    static type = 'output';
-    static version = version;
-    static routes = [
-      {
-        path: '/content',
-        methods: ['get'],
-        handler: 'serve'
+  static type = 'output';
+  static version = version;
+  static routes = [
+    {
+      path: '/dcat-ap/2.0.1',
+      methods: ['get'],
+      handler: 'serve',
+    },
+  ];
+
+  model: any;
+
+  public async serve (req: Request, res: Response) {
+    res.set('Content-Type', 'application/json');
+
+    try {
+      const { domainRecord, siteModel } = await this.fetchDomainAndSite(req.hostname);
+
+      const siteCatalog = _.get(siteModel, 'data.catalog');
+      if (!siteCatalog) {
+        res.status(200).send({});
```

Comment:
Don't you want a `return` here?  Otherwise the function keeps executing after you send a response.

Reply:
I did before, but I replaced it with the `if/else` block which is functionally equivalent. I was on the fence about which was clearer. Happy to change it back if you think it's best.

Reply:
Is it really functionally equivalent?  if you put a `return` here, lines 45 and on won't execute when `siteCatalog` is falsy.  With the if/else, I believe lines 45 and on always execute, regardless of `siteCatalog`.

Reply:
It is, but I think my indentation threw you off. Fixing.

Reply:
Oh, I see it now.  I think this confusion maybe backs up the choice of using `return` rather than if/else.  or maybe just calls into question my eyesight.

I prefer `return`, but will leave it up to you.